src/pages/climbs/[id].tsx

```diff
@@ -295,7 +297,7 @@ const Body = ({ climb, leftClimb, rightClimb }: ClimbPageProps): JSX.Element =>
               </div>
             </div>
 
-            <div className='area-climb-page-summary-right'>
+            <div className='area-climb-page-summary-right lg:h-0 lg:min-h-full'>
```

Comment:
I had a close look at the css last night.  This addition looks a bit hackish if we can avoid it `lg:h-0 lg:min-h-full`, it would be great.  May I suggest the following?
1.  Put the new sibling climbs list JS in a reusable component (can be in the same file as \[id\].tsx)
2. Use tailwind breakpoints to show/hide that component

Example:

```html
<div className='area-climb-page-summary'>
  <div id='Title Information' className='area-climb-page-summary-left'>
    <!-- climb name, metadata, etc ... -->

   <div className='hidden lg:block'> <!-- For desktop: hide it by default, show it when screen >= lg  -->
      <SiblingClimbList {...propsEtc} />
   </div>
  </div>

  <div className='area-climb-page-summary-right'>
    <!-- right pane, description, etc-->
  </div>
</div>

<!-- for mobile -->
<div className='block lg:block'> <!-- the inverse of above, show it by default, hide it when screen  >= lg -->
  <SiblingClimbList {...propsEtc} />
</div>
```

Reply:
We can actually accomplish this with the CSS Grid without duplicating the content. Submitting an commit with a cleaner solution using grid-area

Reply:
Great!  I file a refactoring tasks (lower priority) to consolidate/abstract away the the layout https://github.com/OpenBeta/open-tacos/issues/1066